skyportal/handlers/api/photometry.py

```diff
@@ -1418,12 +1456,51 @@ def put(self):
                                 f'Adding streams {stream_ids_update} to photometry {duplicate.id}'
                             )
 
+                    # if ignoring flux deduplication, do we update the value(s) of the existing photometry?
```

Comment:
Not quite clear what these two lines of comments mean.

Reply:
updated, and as mentioned in another comment this is also explained in the docstring now